q2_demux/plugin_setup.py

```diff
@@ -135,7 +138,9 @@
               'average positional qualities across all of the sequences '
               'selected. If input sequences are paired end, plots will be '
               'generated for both forward and reverse reads for the same `n` '
-              'sequences.')
+              'sequences.'),
+        'delta': ('The acceptable delta cutoff for disparities in sequence '
+                  'length')
```

Comment:
I'm trying to figure out what the data is typically going to look like, and as far as I can find, illumina should always produce uniform lengths unless you've enabled adapter trimming, in which case you're going to get results that vary pretty significantly. So maybe there isn't a good reason for this parameter after all (as you usually won't ever see the warning in the first place).